src/components/DatabaseSizes.tsx

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { li as Li, p as P, table as Table, tbody as Tbody, td as Td, th as Th, thead as Thead, tr as Tr, ul as Ul } from './Mdx';
+
+interface IDatabaseSizes {
+    databasechanges: {
+      csvsizerange: string;
+      databasename: string;
+      ipv4range: string;
+      ipv6range: string;
+      mmdbsizerange: string;
+    }[];
+    daterange: string;
```

Comment:
We should be using camel case here.

```suggestion
    databaseChanges: {
      csvSizeRange: string;
      databaseName: string;
      ipv4Range: string;
      ipv6Range: string;
      mmdbSizeRange: string;
    }[];
    dateRange: string;
```